ignite/cmd/testnet_inplace.go

```diff
@@ -1,10 +1,10 @@
 package ignitecmd
 
 import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-
 	"github.com/spf13/cobra"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
```

Comment:
```suggestion
	sdk "github.com/cosmos/cosmos-sdk/types"
	"github.com/spf13/cobra"

```

Reply:
makes the linter fail

Reply:
But the both imports should group because they is 3-party imports, weird; maybe the others are also wrong

Reply:
No we have the gci linter they are grouped by std External SDK/IBC/COSMOS and Ignite